### DISCUSSION TITLE
Composition plot for three variables

### DISCUSSION THREAD

**Discussion by madbullahi** [david-barnett/microViz#61] [General]
Hello Everyone,
I am trying to plot a relative abundance plot of the microbiome composition for my data which contains three variables. So far, I have made a general plot, but because my data contains is time series data, I would like to visualise the changes in the microbiome composition at different time points. The variables are: 'Genotype', Treatment: Control & Expose and Time: Day1, 2, &3. I have attached a file that shows the plot and an example of the sample-data. I would be grateful if anyone could advise on this.
[example-plot.pptx](https://github.com/david-barnett/microViz/files/9309475/example-plot.pptx)

**Comment by david-barnett** [comment]
Hi, sorry you didn't get a timely reply

Your specific aim remains a little unclear to me, but the new section on sorting or faceting by time might give some inspiration through examples
https://david-barnett.github.io/microViz/articles/web-only/compositions.html#sorting-by-time
